<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAPDBLibTest</class>
 <widget class="QMainWindow" name="SAPDBLibTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAPDBLibTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 245, 245);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="gbSapFromPa">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>481</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SAP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FlashDB-dev</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSapFromPa">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SapFromPa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbQtyPass">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>130</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>QtyPass</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbGesAufMenge">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>130</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbMaterialNr">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbGesAufMengeFlash">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>130</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbMaterialNrFlash">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbGesMenAuftrag">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Gesamtmenge Auftrag</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbMaterialNr">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MaterialNr</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbQtyPass">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>481</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbOrderType">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>210</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>getOrderType</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbOrderType">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>180</y>
       <width>481</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <widget class="QTextBrowser" name="tbOrderType">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>91</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;60090101 - error&lt;/p&gt;&lt;p&gt;60090102 - error&lt;/p&gt;&lt;p&gt;60095542 - error&lt;/p&gt;&lt;p&gt;60090103 - ok&lt;/p&gt;&lt;p&gt;60098052 - ok&lt;/p&gt;&lt;p&gt;60091935 - ok&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>231</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(204, 204, 204);</string>
     </property>
     <property name="title">
      <string>Eingabe</string>
     </property>
     <widget class="QLabel" name="lbAuftragsNr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Auftragsnummer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAuftragsNr">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>60091935</string>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_4</zorder>
    <zorder>gbQtyPass</zorder>
    <zorder>gbSapFromPa</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>pbSapFromPa</zorder>
    <zorder>pbQtyPass</zorder>
    <zorder>tbGesAufMenge</zorder>
    <zorder>tbMaterialNr</zorder>
    <zorder>tbGesAufMengeFlash</zorder>
    <zorder>tbMaterialNrFlash</zorder>
    <zorder>lbGesMenAuftrag</zorder>
    <zorder>lbMaterialNr</zorder>
    <zorder>pbOrderType</zorder>
    <zorder>gbOrderType</zorder>
    <zorder>label_10</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>1031</width>
      <height>231</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777212</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(245, 245, 245);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SAP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbReadMatKurzText">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>getMKT</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbMaterialkurztext">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Materialkurztext</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>221</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;602663 - ok&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;602947 - ok&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;601783 - ok&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;601996 - ok&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;602250 - error&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;602760 - error&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;601995 - error&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;601997 - error&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;601998 - error&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;601999 - error&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>FlashDB-dev</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbGetMKT">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>491</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbMaterialkurztextFlash">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbMaterialkurztext">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(204, 204, 204);</string>
     </property>
     <property name="title">
      <string>Eingabe</string>
     </property>
     <widget class="QLabel" name="lbAuftragsNr_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SAP Nr</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSapNr">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>604424</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>391</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 245, 245);</string>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="tbStatus">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>150</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPing">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>150</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping</string>
     </property>
    </widget>
    <zorder>groupBox_3</zorder>
    <zorder>label_4</zorder>
    <zorder>pbReadMatKurzText</zorder>
    <zorder>lbMaterialkurztext</zorder>
    <zorder>label_5</zorder>
    <zorder>label_9</zorder>
    <zorder>gbGetMKT</zorder>
    <zorder>tbMaterialkurztextFlash</zorder>
    <zorder>tbMaterialkurztext</zorder>
    <zorder>groupBox_5</zorder>
    <zorder>tbStatus</zorder>
    <zorder>pbPing</zorder>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1037</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
